lib/features/authentication/presentation/providers/bottom_provider.dart

```diff
@@ -0,0 +1,3 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+final pageProvider = StateProvider((ref) => 0);
```

Comment:
Create a home feature

Addd the widget into the core feature

provider should be added to the home feature